### PR TITLE
Update java key vault names

### PR DESCRIPTION
**Pull request by jongio**

### CODE REVIEW

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -38,12 +38,12 @@
 "azure-ai-formrecognizer","com.azure","3.0.2","","Form Recognizer","Form Recognizer","formrecognizer","","","client","true","",""
 "azure-identity","com.azure","1.1.3","1.2.0-beta.2","Identity","Identity","identity","","","client","true","",""
 "azure-security-keyvault-administration","com.azure","","4.0.0-beta.2","Key Vault - Administration","Key Vault","keyvault","","","client","true","",""
-"azure-keyvault-certificates","com.azure","","4.0.0-preview.4","Key Vault - Certificates","Key Vault","keyvault","NA","NA","client","true","","Replaced by: azure-security-keyvault-certificates"
+"azure-keyvault-certificates","com.azure","","4.0.0-preview.4","Key Vault - Certificates","Key Vault","keyvault","NA","NA","client","false","","Replaced by: azure-security-keyvault-certificates"
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:41]: We might also want to consider hiding these packages.

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (jongio) [reply _data/releases/latest/java-packages.csv:37]: Which packages?  It didn't come through in the PR highlight.

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:37]: I think all these older keyvault packages.

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -135,7 +129,7 @@
 "azure-cognitiveservices-autosuggest","com.microsoft.azure.cognitiveservices","","1.0.2-beta","Autosuggest","Autosuggest","https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cognitiveservices/ms-azure-cs-autosuggest","NA","NA","client","false","",""
 "azure","com.microsoft.azure","1.37.0","","Azure","Azure","NA","NA","NA","client","false","",""
 "azure-active-directory-b2c-spring-boot-starter","com.microsoft.azure","2.3.5","","Azure Active Directory - B2C Spring Boot Starter","Azure Active Directory","https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory-b2c","","NA","client","false","",""
-"azure-ad-integration-spring-boot-autoconfigure","com.microsoft.azure","0.1.5","","Azure Active Directory - Integration Spring Boot Autoconfigure","Azure Active Directory","NA","NA","NA","client","false","",""
+"azure-ad-integration-spring-boot-autoconfigure","com.microsoft.azure","0.1.5","","s","Azure Active Directory","NA","NA","NA","client","false","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:132]: Typo here?

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:37]: Also your dashboard might want to ignore anything with Hide=true as well, if it isn't already.

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
broken links aren't related to these changes so merging.

**Pull request merged**